view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -0,0 +1,184 @@
+        <!-- ko template: getTemplate() -->
+        <!-- /ko -->
+        <!--/ko-->
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() -->
+            <!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="omise-banking-fee-warn" data-bind="i18n: 'Choose your installment terms'"></div>
+        <form class="form" data-bind="attr: {
+              id: getCode() + 'Form',
+              }">
+
+            <fieldset data-bind="attr: {class: 'fieldset payment items ' + getCode(), id: 'payment_form_' + getCode()}">
+                <ul class="form-list">
+                    <!-- KTC -->
+                    <li class="items">
+                        <input type="radio" name="offsite" value="installment_ktc" data-bind="
+                            attr: {
+                                id: getCode() + '_ktc'
+                            },
+                            checked: omiseOffsite,
+                            click: function(data, event) { 
+                                    installmentTerms(null); 
+                                    return true; 
+                                }" />
+                        <label for="omise_offsite_installment_ktc">
+                            <div class="omise-logo-wrapper ktc">
+                                <img class="ktc" />
+                            </div>
+                            <div class="omise-banking-text-wrapper">
+                                <span class="title" data-bind="i18n: 'Krungthai Card'"></span>
+                            </div>
+                        </label>
+                        <select style="margin-top:10px" data-bind="options: getInstallmentTerms('ktc'),
+                                        optionsText: function(item) {
+                                            return item;
+                                        },
+                                        valueAllowUnset: true,
+                                        value: installmentTerms,
+                                        visible: omiseOffsite() === 'installment_ktc',
+                                        optionsCaption: 'Choose terms...'"></select>
+                    </li>
+
+                    <!-- First Choice -->
+                    <li class="items">
+                        <input type="radio" name="offsite" value="installment_first_choice" data-bind="
+                            attr: {
+                                id: getCode() + '_first_choice'
+                            },
+                            checked: omiseOffsite,
+                            click: function(data, event) { 
+                                installmentTerms(null); 
+                                return true; 
+                            };" />
+                        <label for="omise_offsite_installment_fc">
```

Comment:
@jacstn `for` attribute has different value with the `<input>` element (`omise_offsite_installment_fc` & `getCode() + '_first_choice'`)

Reply:
Would be fixed by not hardcoding it, bringing the names from the `backend` data

Reply:
@jacstn in case you missed this one